Tests/.swiftlint.yml

```diff
@@ -0,0 +1,12 @@
+disabled_rules:
+  - line_length
+  - type_name
+  - valid_docs
+  - force_unwrapping
+  - force_cast
+  - force_try
+  - function_body_length
+  - cyclomatic_complexity
```

Comment:
And this one because we have many workaround that we need to do for testing, so we would also have a lot of comments disabling it.